### ISSUE TITLE
Powershell\initializeAKS.ps1 script was giving an error like "az: error: unrecognized arguments: --enable-managed-identity --aks-custom-headers=CustomizedUbuntu=aks-ubuntu-1804,ContainerRuntime=containerd --attach-acr=ngAcrRegistry2"

### ISSUE THREAD

**Issue by tutul2010** [NileshGule/pd-tech-fest-2019#46]
I am using windows-10-Pro, 64 bit and windows power-shell versions as below for dev environment...

```
Name                           Value
----                           -----
PSVersion                      5.1.19041.610
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.610
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Also, I am new to power-shell and tried to execute initializeAKS.ps1 for provisioning an AKS infrastructure but it was failing with an error like 
**"az: error: unrecognized arguments: --enable-managed-identity --aks-custom-headers=CustomizedUbuntu=aks-ubuntu-1804,ContainerRuntime=containerd --attach-acr=ngAcrRegistry2"**
Below mentioned script's code-block was giving an  exception .. _Pls help or suggest to enabling managed-identity  with AKS .._

```
if ($aksCLusterExists -eq $false) {
    # Create AKS cluster
    Write-Host "Creating AKS cluster $clusterName with resource group $resourceGroupName in region $resourceGroupLocaltion" -ForegroundColor Yellow
    az aks create `
        --resource-group=$resourceGroupName `
        --name=$clusterName `
        --node-count=$workerNodeCount `
        --enable-managed-identity `
        --output=jsonc `
        --kubernetes-version=$kubernetesVersion `
        --aks-custom-headers="CustomizedUbuntu=aks-ubuntu-1804,ContainerRuntime=containerd" `
        --attach-acr=$acrRegistryName '

} 
```
Pls, Any suggestions..

Many Thanks,

**Comment by tutul2010** [comment]
Resolved by  updating  Azure CLI version 2.15.1 or later at dev system(windows -10 pro ).

**Comment by NileshGule** [comment]
The issue has been closed by upgrading the az cli version
